src/main.cpp

```diff
@@ -7170,7 +7170,7 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             return false;
         }
 
-		if (pfrom->nVersion < 180322 & !fTestNet && pindexBest->nHeight > 855000)
+		if (pfrom->nVersion < 180322 && !fTestNet && pindexBest->nHeight > 855000)

```

Comment:
This one is a typo, but if you expand it it just happens to work out. It's merely a nuisance :)